.travis.yml

```diff
@@ -56,7 +56,7 @@ env:
     - ROS_DISTRO=kinetic OS_NAME=debian OS_CODE_NAME=xenial EXPECT_EXIT_CODE=1
     - ROS_DISTRO=kinetic DOCKER_BASE_IMAGE="ros:kinetic-ros-base" NOT_TEST_BUILD='true' DEBUG_BASH='true' VERBOSE_OUTPUT='false'
     - ROS_DISTRO=kinetic APTKEY_STORE_SKS=hkp://ha.pool.sks-keyservers.vet  # Passing wrong SKS URL as a break test. Should still pass.
-    - ROS_DISTRO=lunar ROS_REPO=ros-shadow-fixed USE_MOCKUP='industrial_ci/mockups/industrial_ci_testpkg'
+    - ROS_DISTRO=lunar ROS_REPO=ros-shadow-fixed USE_MOCKUP='industrial_ci/mockups/industrial_ci_testpkg' CATKIN_LINT=true
```

Comment:
`industrial_ci_testpkg` should get patched to issue a warning, so we can add a supposed-to-fail test using the `--strict' flag.

Reply:
Do you mean we should use something else, not `industrial_ci_testpkg` for checking catkin_lint feature? If so do you have a suggestion?

Reply:
> Do you mean we should use something else, not industrial_ci_testpkg for checking catkin_lint feature? If so do you have a suggestion?

No, that's fine.
But I want to add another teststhat fails on purpose to actually that `catkin_lint` was run for the given situation.